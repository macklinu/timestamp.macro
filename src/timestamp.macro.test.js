import pluginTester from 'babel-plugin-tester'
import plugin from 'babel-plugin-macros'
import prettier from 'prettier'
import prettierOptions from 'best-prettier-config'

let format = code => prettier.format(code, prettierOptions)

pluginTester({
  plugin,
  pluginName: 'timestamp.macro',
  snapshot: true,
  babelOptions: { filename: __filename },
  tests: [
    format(`
      import timestamp from './timestamp.macro'
      let value = timestamp('2018-10-01')
    `),
  ],
})
