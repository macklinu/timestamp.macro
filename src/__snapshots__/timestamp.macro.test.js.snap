// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timestamp.macro 1. timestamp.macro: 1. timestamp.macro 1`] = `
"
import timestamp from './timestamp.macro'
let value = timestamp('2018-10-01')

      ↓ ↓ ↓ ↓ ↓ ↓

let value = timestamp('2018-10-01');
"
`;
